index.html

```diff
@@ -97,13 +97,13 @@
 <body>
   <section id="abstract">
     <p>
-      The present document defines a WoT Profile that enables out of the box interoperability
-      among things and devices. It defines a set of constraints and rules that compliant
-      thing descriptions have to adopt to ensure interoperability.
+        This document introduce a WoT profile concept that supports the so-called <i>out of the box interoperability</i>
```

Comment:
This language is less clear than the previous text and waters down the purpose of the core profile.

Reply:
I would say this is the same content. However, I changed the word "introduces" to "defines". What do you think?